src/Maps/components/MapsContainer.ts

```diff
@@ -0,0 +1,242 @@
+import { Component, createElement } from "react";
+import { LeafletEvent } from "leaflet";
+
+import { LeafletMap } from "./LeafletMap";
+import googleApiWrapper from "./GoogleMap";
+import { Container } from "../utils/namespace";
+import { fetchData, fetchMarkerObjectUrl, parseStaticLocations } from "../utils/Data";
+import { validLocations, validateLocationProps } from "../utils/Validations";
+import MapsContainerProps = Container.MapsContainerProps;
+import MapProps = Container.MapProps;
+import Location = Container.Location;
+import DataSourceLocationProps = Container.DataSourceLocationProps;
+
+// tslint:disable-next-line:no-submodule-imports
```

Comment:
Just disable this in your `tslintConfig` file

Reply:
Then remove these lines

Reply:
done